src/Command/PasetoCommand.php

```diff
@@ -0,0 +1,96 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * CakePHP(tm) : Rapid Development Framework (https://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ * @link          https://cakephp.org CakePHP(tm) Project
+ * @since         3.0.0
+ * @license       https://opensource.org/licenses/mit-license.php MIT License
+ */
+namespace Authentication\Command;
+
+use Authentication\Authenticator\PasetoAuthenticator;
+use Cake\Command\Command;
+use Cake\Console\Arguments;
+use Cake\Console\ConsoleIo;
+use Cake\Console\ConsoleOptionParser;
+use ParagonIE\Paseto\Keys\AsymmetricSecretKey;
+use ParagonIE\Paseto\Keys\SymmetricKey;
+use ParagonIE\Paseto\Protocol\Version3;
+use ParagonIE\Paseto\Protocol\Version4;
+use ParagonIE\Paseto\ProtocolInterface;
+
+class PasetoCommand extends Command
+{
+    /**
+     * @param \Cake\Console\ConsoleOptionParser $parser ConsoleOptionParser
+     * @return \Cake\Console\ConsoleOptionParser
+     */
+    protected function buildOptionParser(ConsoleOptionParser $parser): ConsoleOptionParser
+    {
+        $parser
+            ->setDescription('Generate keys for PASETO')
+            ->addArgument('version', [
+                'help' => 'The PASETO version',
+                'required' => true,
+                'choices' => ['v3', 'v4'],
+            ])
+            ->addArgument('purpose', [
+                'help' => 'The PASETO purpose',
+                'required' => true,
+                'choices' => [PasetoAuthenticator::LOCAL, PasetoAuthenticator::PUBLIC],
```

Comment:
Would be good to have a default value on these too. Good defaults help users succeed with less effort.

Reply:
We can set defaults to v4 local here as well.